creator-node/src/utils/fileUtils.ts

```diff
@@ -0,0 +1,16 @@
+import isMp3 from 'is-mp3'
+
+const readChunk = require('read-chunk')
+
+const ffmpegCharCodes = Array.from('FFmpeg').map(s => s.charCodeAt(0))
+
+export const isMp3File = async (path: string) => {
```

Comment:
Should we try/catch this? what happens if we try to read from a file with length < 25 bytes?

Reply:
assuming that the 'FFmpeg' bytes should always be there, I don't think we'll hit that scenario. but better to be safe